.gitignore

```diff
@@ -1,3 +1,6 @@
+.idea
+.vscode
+
```

Comment:
```suggestion
.vscode
```

Reply:
@MikeMcQuaid done:)

This PR contains only zsh-completion script

`dump` command logic updated in https://github.com/Homebrew/homebrew-bundle/pull/1217 and should be reviewed and merged after this PR, thanks:)